components/RoomDetailsPage/components/MainContent/MainContent.tsx

```diff
@@ -7,49 +9,54 @@ import RoomImpression from 'components/RoomImpression/RoomImpression';
 import { roomImagesPreview, benefitsData, rulesData } from './MainContent.data';
 import * as S from './MainContent.styles';
 
-const MainContent = (): JSX.Element => (
-  <S.MainContent>
-    <S.RoomImages>
-      {roomImagesPreview.map(({ src, alt }) => (
-        <img key={src} src={src} alt={alt} />
-      ))}
-    </S.RoomImages>
-    <S.Details>
-      <S.Benefits>
-        <S.Title>Сведения о номере</S.Title>
-        <Benefits items={benefitsData} />
-      </S.Benefits>
-      <S.RoomImpressionWrapper>
-        <RoomImpression
-          title="Впечатления от номера"
-          numberOfRatings={{ excellent: 130, good: 65, normal: 65, bad: 0 }}
-        />
-      </S.RoomImpressionWrapper>
-      <S.ReviewsWrapper>
-        <Reviews />
-      </S.ReviewsWrapper>
-      <S.BulletList>
-        <S.Title>Правила</S.Title>
-        <BulletList items={rulesData} />
-      </S.BulletList>
-      <S.CancellationTerms>
-        <S.Title>Отмена</S.Title>
-        <S.CancellationTermsText>
-          Бесплатная отмена в течение 48 ч. После этого при отмене не позднее чем за 5 дн. до
-          прибытия вы получите полный возврат за вычетом сбора за услуги.
-        </S.CancellationTermsText>
-      </S.CancellationTerms>
-      <S.OrderFormWrapper>
-        <OrderForm
-          overcrowdingPrice={700}
-          breakfastPricePerGuest={300}
-          roomNumber={888}
-          roomType="люкс"
-          roomPrice={9990}
-        />
-      </S.OrderFormWrapper>
-    </S.Details>
-  </S.MainContent>
-);
+const MainContent = (): JSX.Element => {
+  const { t } = useTranslation('MainContent');
+
+  return (
+    <S.MainContent>
+      <S.RoomImages>
+        {roomImagesPreview.map(({ src, alt }) => (
+          <img key={src} src={src} alt={alt} />
+        ))}
+      </S.RoomImages>
+      <S.Details>
+        <S.Benefits>
+          <S.Title>{t('Room details')}</S.Title>
+          <Benefits items={benefitsData} />
+        </S.Benefits>
+        <S.RoomImpressionWrapper>
+          <RoomImpression
+            title={t('Impressions of the room')}
+            numberOfRatings={{ excellent: 130, good: 65, normal: 65, bad: 0 }}
+          />
+        </S.RoomImpressionWrapper>
+        <S.ReviewsWrapper>
+          <Reviews />
+        </S.ReviewsWrapper>
+        <S.BulletList>
+          <S.Title>{t('Rules')}</S.Title>
+          <BulletList items={rulesData} />
+        </S.BulletList>
+        <S.CancellationTerms>
+          <S.Title>{t('Cancel')}</S.Title>
+          <S.CancellationTermsText>
+            {t(
+              'Free cancellation within 48 hours. Thereafter, if canceled no later than 5 days in advance. before upon arrival, you will receive a full refund minus the service fee.',
```

Comment:
Thereafter you will get a full refund minus the service fee if the reservation is canceled 5 days or more prior to the arrival date